backend/src/test/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceControllerTest.java

```diff
@@ -167,6 +177,161 @@ void verifyLinkV2ReturnsTestResultInfo() throws Exception {
     assertLastAuditEntry(HttpStatus.OK, ResourceLinks.VERIFY_LINK_V2, requestId);
   }
 
+  @Test
+  void verifyLinkV2ReturnsTestResultInfo_multiplex() throws Exception {
+
+    TestUserIdentities.withStandardUser(
+        () -> {
+          testEvent =
+              _dataFactory.createMultiplexTestEvent(
+                  person,
+                  facility,
+                  TestResult.POSITIVE,
+                  TestResult.NEGATIVE,
+                  TestResult.NEGATIVE,
+                  false);
+          patientLink = _dataFactory.createPatientLink(testEvent.getTestOrder());
+        });
+
+    Set<SupportedDiseaseTestResult> multiplexResults = new HashSet<>();
+    multiplexResults.add(
+        new SupportedDiseaseTestResult(_diseaseService.covid(), TestResult.POSITIVE));
+    multiplexResults.add(
+        new SupportedDiseaseTestResult(_diseaseService.fluA(), TestResult.NEGATIVE));
+    multiplexResults.add(
+        new SupportedDiseaseTestResult(_diseaseService.fluB(), TestResult.NEGATIVE));
+
+    ObjectMapper mapper = new ObjectMapper();
+    String multiplexResponse = mapper.writeValueAsString(multiplexResults);
+    JSONObject multiplexResponseJson = new JSONObject("results", multiplexResponse);
+
+    // GIVEN
+    String dob = person.getBirthDate().format(DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+    String requestBody =
+        "{\"patientLinkId\":\""
+            + patientLink.getInternalId()
+            + "\",\"dateOfBirth\":\""
+            + dob
+            + "\"}";
+
+    // WHEN
+    MockHttpServletRequestBuilder builder =
+        post(ResourceLinks.VERIFY_LINK_V2)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .accept(MediaType.APPLICATION_JSON)
+            .characterEncoding("UTF-8")
+            .content(requestBody);
+
+    // THEN
+    String requestId =
+        mockMvc
+            .perform(builder)
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.testEventId", is(testEvent.getInternalId().toString())))
+            .andExpect(jsonPath("$.results", Matchers.hasSize(3)))
+            .andExpect(
+                jsonPath(
+                    "$.results[0]",
+                    Matchers.hasItemInArray(
+                        mapper.writeValueAsString(
+                            new SupportedDiseaseTestResult(
+                                _diseaseService.fluB(), TestResult.NEGATIVE)))))
+            //                    .andExpect(jsonPath("$.results",
```

Comment:
I tried approximately one million variations of this assertion to get it to pass, with no luck - `containsInAnyOrder`, `contains`, `in`, `hasItemInArray`, matching individual elements vs the entire object, etc. Does anyone have experience with these matchers and know how to assert on this correctly?